docs/en/stack/ml/get-started/ml-gs-results.asciidoc

```diff
@@ -50,7 +50,8 @@ display only a single time series.
 The blue line in the chart represents the actual data values. The shaded blue
 area represents the bounds for the expected values. The area between the upper
 and lower bounds are the most likely values for the model. If a value is outside
-of this area then it can be said to be anomalous.
+of this area then it can be said to be anomalous. The shaded blue, therefore, is
```

Comment:
If my two earlier suggestions are accepted, this final sentence is no longer required:
```suggestion
of this area then it can be said to be anomalous.
```